### PR TITLE
Enhanced Dealer Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#24057 /bcgov/entity#24017 /bcgov/entity#24243

*Description of changes:*
- Admin and User versions of Enhanced Dealer Information Component
- Conditionally bypassing Dealer Transport Permit and Transfer Restrictions
- Update SBC Permissions on Transport Permits

Screenshot 2024-11-06 at 11 52 35 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/userAccess/DealerInfo.vue [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+const { getIsAccountAdministrator } = storeToRefs(useStore())
+const { downloadServiceAgreement } = useUserAccess()
+
+const activePanels = ref<number[]>([0])
+const qsUserGuideUrl = 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/' +
+  'business-management/permits-licences-and-registration/registries-other-assets/qs_transfers_sale_user_guide.pdf'
+
+const acceptTerms = () => {
+  console.log('API PUT accepted agreement terms here')
+}
```
Comment (cameron-eyds) [inline ppr-ui/src/components/userAccess/DealerInfo.vue:16]: To be replaced with PUT in ticket #24273

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-2058-znux19at.web.app

**Review by flutistar — APPROVED** [review]
LGTM

**Pull request merged**